CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.11.0 (Unreleased)
+## 0.11.1
```

Comment:
0.11.0 got released for Python and Kotlin, so we should probably keep that section around and just add a new one for 0.11.1

Reply:
It did not get released for Kotlin, which was part of my confusion here.

Reply:
That is odd... It shows [here](https://github.com/IronCoreLabs/ironcore-alloy/actions/runs/9070025976/job/24921043859) that the Kotlin release succeeded. Maybe we forgot to do the `oss.sonatype.org` step so it didn't fully release? I just checked though and it's not in Staging Repositories still

Reply:
Yeah I checked it with Colt the other day and saw it wasn't released or in staging. With this `0.11.1` _should_ go out for Rust, Python, Java, and Kotlin (as long as dev dependencies can be git for Rust), which'll be the first release of these features for all but Python.

Otherwise I could make a `0.11.0 (Python)` entry with these features, then a `0.11.1` with the same text and Java for all the others or something?

Reply:
Yeah since it was only Python that actually got it I'm good with how you have it